subprojects/model-core/src/testFixtures/groovy/org/gradle/api/internal/provider/ProviderTestUtil.java

```diff
@@ -24,6 +24,7 @@
 import java.util.Iterator;
 import java.util.List;
 
+@SuppressWarnings("unchecked")
```

Comment:
Which warning is being suppressed here?

Reply:
I guess it has something to do with the generic varargs.

Reply:
Can you use `@SafeVargs` to get rid of the deprecation message?

Reply:
I'm removing this as it's not a problem with Groovy 4 either.